src/components/settings/SettingsTabs.jsx

```diff
@@ -40,12 +41,12 @@ const SettingsTabs = () => {
   };
 
   return (
-    <div className="m-4">
+    <Container className="py-4" fluid>
```

Comment:
Not sure why we use `Container` over just a `div` here? Is it for the classes that come with `fluid`?

Reply:
Yeah, I noticed the spacing was a bit off with the previous use of `Container`. By moving the `Container` to `SettingsTabs` instead of where it was previously, the spacing is more in line with the containers for other pages with regard to left/right padding.